### ISSUE TITLE
Mapping themes for popular agencies

### ISSUE THREAD

**Issue by dirkschumacher** [reconhub/hackfest2#1]
A bit basic, but probably useful. Themes/Tutorials/Functions for maps that make it easier to create maps in a similar style as published by major UN agencies and NGOs. This might be useful to compete with the complex maps produced by hand in progams like ArcGIS.

WHO Ebola Maps: http://www.who.int/csr/disease/ebola/maps-2014/en/
UN OCHA: https://docs.unocha.org/sites/dms/SouthSudan/2017_SouthSudan/170529_Cholera_Outbreak.gif

**Comment by Paula-Moraga** [comment]
I agree ArcGIS is very good for making maps. It would be great if we can find R packages for doing similar things.

**Comment by thibautjombart** [comment]
Yes +1 to this, definitely. We could also have our RECON themes ;)
